gammapy/estimators/utils.py

```diff
@@ -753,3 +754,80 @@ def get_combined_significance_maps(estimator, datasets):
         npred_excess=npred_excess_sum,
         estimator_results=results,
     )
+
+
+def joint_flux_maps(maps, method="gaussian_errors", **kwargs):
+    """Create a FluxMaps by combining a list of flux maps with the same geometry.
+     This assumes the flux maps are independent measurements of the same true value.
+
+    Parameters
+    ----------
+    maps : list of `~gammapy.estimators.FluxMaps` or ~gammapy.estimators.FluxMaps`
+        List of maps with the same geometry.
+    kwargs : dict
+        Options passed to `~gammapy.estimators.FluxMaps.from_maps`
```

Comment:
Maybe the `kwargs` here is a bit misleading because one can't pass a `gti`, and the `sed_type` will always be overwritten. 
Does one need to pass a reference model separately? The individual `FluxMaps` should have a `reference_model` contained. 
What is the new reference model? Does this only work for the same `reference_model` for all maps?

So what is passed through `kwargs` is mainly the meta?

Reply:
Now the `reference_model` is added to the arguments list instead of `kwargs` (there is no issues using a different reference model).
The meta is added taking the info from the parent FluxMaps (in a very simple way) but we will have to come back to this later when `FluxMapsMetaData.stack` and `FluxPointsMetaData.stack` are implemented.

Reply:
Yes, we can keep the meta for later. It currently stores the `n_sigma_ul` which should be the same between the different maps I think.